doc/classes/TextEdit.xml

```diff
@@ -101,6 +101,12 @@
 				Adjust the viewport so the caret is visible.
 			</description>
 		</method>
+		<method name="apply_ime">
+			<return type="void" />
+			<description>
+				Applies text from the IME to each caret and closes the IME if it is open.
```

Comment:
It's not really explained what IME stands for.
```suggestion
				Applies text from the Input Method Editor (IME) to each caret and closes the IME if it is open.
```
Or, should we have a link?
```suggestion
				Applies text from the [url=https://en.wikipedia.org/wiki/Input_method]Input Method Editor[/url] (IME) to each caret and closes the IME if it is open.
```

Reply:
I'll use the link so it matches the DisplayServer docs.